docs/application/toc_all.md

```diff
@@ -1173,3 +1173,10 @@
 ## [Prerequisites](/application/vscode-ext/index.md)
 ## [Tizen .NET](/application/vscode-ext/dotnet.md)
 ## [Tizen Web](/application/vscode-ext/web.md)
+
+
+# Visual Studio Code Extension for Mac
+
```

Comment:
It would be great if there is an overview paragraph.
It is not recommended that there is no phrase between section and section.

Reply:
I think this comment is in the wrong position.

A simple sentence is added to the overview.md file.